composer.json

```diff
@@ -49,11 +49,11 @@
 		"mediawiki/semantic-media-wiki": "~3.0",
 		"nicmart/tree": "^0.2.7",
 		"data-values/geo": "~4.0|~3.0|~2.0",
-		"phpoffice/phpspreadsheet": "^1.4",
 		"symfony/css-selector": "^3.3",
 		"mediawiki/mermaid": "~2.1"
 	},
 	"suggest": {
+		"phpoffice/phpspreadsheet": "Required for 'format=spreadsheet'",
```

Comment:
This does not work since "spreadsheet" is enabled by default: https://github.com/SemanticMediaWiki/SemanticResultFormats/blob/master/DefaultSettings.php#L55